integration/spark/app/src/test/java/io/openlineage/spark/agent/SparkContainerIntegrationTest.java

```diff
@@ -122,6 +122,20 @@ void testPysparkRddToTable() {
         "pysparkRddToTableCompleteEvent.json");
   }
 
+  @Test
+  @EnabledIfSystemProperty(named = SPARK_VERSION, matches = SPARK_3) // Spark version >= 3.*
+  void testJdbcColumnDbtable() {
+    GenericContainer mysql = SparkContainerUtils.makeMysqlContainer(network);
+    try {
+      mysql.start();
```

Comment:
Wouldn't it be better to create separate test suite like `SparkJdbcIntegrationTest` to have mysql docker shared for all such tests? 

Wouldn't it be better to run Spark in memory and access spark session within test method instead running whole docker? Such tests are a way faster than running spark in docker and do not require mock server container.

Reply:
Rewrote to Spark-in-memory test.